--- a/madgraph/various/banner.py
+++ b/madgraph/various/banner.py
@@ -2736,7 +2736,7 @@ class RunCardLO(RunCard):
 #**************************************
 #
 %(sys_scalefact)s = sys_scalefact  # factorization/renormalization scale factor
-%(sys_alpsfact)s = sys_alpsfact  # \alpha_s emission scale factors
+%(sys_alpsfact)s = sys_alpsfact  # alpha_s emission scale factors
 %(sys_matchscale)s = sys_matchscale # variation of merging scale
 # PDF sets and number of members (0 or none for all members).
 %(sys_pdf)s = sys_pdf # list of pdf sets. (errorset not valid for syscalc)
